src/common/meta/src/datanode.rs

```diff
@@ -77,7 +77,7 @@ pub struct RegionStat {
     pub rcus: i64,
     /// The write capacity units during this period
     pub wcus: i64,
-    /// Approximate bytes of this region
+    /// Approximate disk bytes of this region, including sst, index, manifest and wal
```

Comment:
Maybe the data size will be a better word. We store data on S3 or disk.

Reply:
That's acceptable. Just a hint.